src/clib/lib/enkf/ensemble_config.cpp

```diff
@@ -151,7 +149,13 @@ enkf_config_node_type *ensemble_config_get_or_create_summary_node(
 
 bool ensemble_config_have_forward_init(
     const ensemble_config_type *ensemble_config) {
-    return ensemble_config->have_forward_init;
+
+    for (auto const &node_it : ensemble_config->config_nodes) {
+        if (enkf_config_node_use_forward_init(node_it.second))
```

Comment:
ok just two small questions here -
- in the naming `node_it`, what does the `it` signify wait while writing this i figured out it stands for iterator would you mind spelling it out for the slow ones among us? : D
- the .second throws me off a bit, after checking the type of the `config_nodes` field it becomes clear though. would it be weird to add something like `tuple` to the name of the iterator variable?

Reply:
```
        for (auto const &[str, enkf_config_node] : self->config_nodes) {
            if (enkf_config_node->forward_init)
```
Will this work? We use C++17, so this should be syntactically ok.

Reply:
Updated according to slack discussion.

Reply:
i'm sorry i have no idea - i'm not very familiar with the differences between C++ standards, and don't know which one we're using. try and push, see if it doesn't fail in CI?

Reply:
This syntax require C++17, it builds locally, so i guess this will be alright. Jenkins has the last word regardless.